e2etests/web/text_editing/build/flutter_assets/AssetManifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
The `build` directory should not be checked in

Reply:
done.